hw/hw11/mailroom.py

```diff
@@ -0,0 +1,151 @@
+doners = [['John Smith', 320],
+          ['Mary Simpson', 100],
+          ['Chris Finch', 400],
+          ['Larry Bookman', 150],
+          ['Victoria Black', 60],
+          ['Mary Simpson', 150],
+          ['Mary Simpson', 200],
+          ['Larry Bookman', 200]]
+
```

Comment:
When defining global variables in python, it is standard to declare them in all caps, so: DONERS